tendrl/commons/objects/cluster/atoms/setup_cluster_alias/__init__.py

```diff
@@ -35,22 +35,31 @@ def run(self):
         wait_count = 24
         while True:
             child_job_failed = False
+            job = Job(job_id=_job_id).load()
             if loop_count >= wait_count:
+                if job.status in ["new", ""]:
+                    msg = ("Child job %s for setting up cluster alias not yet "
+                           "picked up by the server, Service "
+                           "tendrl-monitoring-integration may be down. "
+                           "Timing out. (%s)" % (
+                               job.job_id, integration_id
+                           ))
+                else:
+                    msg = ("Child job %s for setting up cluster alias not yet"
+                           "compeleted by the server. Timing out. (%s)" % (
+                               job.job_id, integration_id))
                 logger.log(
                     "error",
                     NS.publisher_id,
                     {
-                        "message": "Setting up cluster alias"
-                        "not yet complete. Timing out. (%s)" %
-                        integration_id
+                        "message": msg
                     },
                     job_id=self.parameters['job_id'],
                     flow_id=self.parameters['flow_id'],
                 )
                 return False
             time.sleep(5)
             finished = True
-            job = Job(job_id=_job_id).load()
```

Comment:
It should be loaded again for job

Reply:
yes, it will create and populate the fields in new object